### PR TITLE
Added Hacktoberfest and Strimzi blog post

### PR DESCRIPTION
**Pull request by ppatierno**
This PR adds a short blog post about Hacktoberfest starting and the opportunity to contribute to Strimzi.

### CODE REVIEW

File: _posts/2020-10-03-hacktoberfest.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "Hacktoberfest: let's start to contribute to Strimzi"
+date: 2020-10-03
```
Comment (scholzj) [inline _posts/2020-10-03-hacktoberfest.md:4]: Shouldn't we publish it tomorrow on _1st_ given how it ties to October?

File: _posts/2020-10-03-hacktoberfest.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Maybe you are asking to yourself, "why talking about Hacktoberfest on the Strimzi website?"
+It is simple! Right?
+
+The Strimzi community would be really happy to take part at this event and having more people to get involved with contributions.
+After the last blog post, about the first [Strimzi survey results](https://strimzi.io/blog/2020/09/29/strimzi-survey-results/), we noticed that developers would love to contribute to Strimzi and they have planned to do so.
+So, what is stopping you?
+What could be a better opportunity than starting with the Hacktoberfest?
+
+Frome simpler to more complex ones, from code to documentation, you can find different [issues](https://github.com/strimzi/strimzi-kafka-operator/issues) on our GitHub repository in order to engage with us and starting to contribute.
+If you don't find something that can fit your knowledge or you have any kind of problems and need advices, don't hesitate to ask us anything in the Slack room or on mailing list.
```
Comment (scholzj) [inline _posts/2020-10-03-hacktoberfest.md:31]: I think you should go into more details here and point directly to some of the labels:
* https://github.com/strimzi/strimzi-kafka-operator/labels/good-start
* https://github.com/strimzi/strimzi-kafka-operator/labels/documentation
* maybe https://github.com/strimzi/strimzi-kafka-operator/labels/help%20wanted

That will help people to orient them self faster and find the right issue.

MAybe we could also ask them to comment on the issue if they start working on it to allow multiple people working the same issue?

File: _posts/2020-10-03-hacktoberfest.md (hunk 1 above)
Comment (ppatierno) [reply _posts/2020-10-03-hacktoberfest.md:4]: I agree, I was not sure about the time needed to review :-) I will change it.

**Review by scholzj — APPROVED** [review]

**Comment by sjwoodman** [comment]
LGTM - thanks Paolo!

**Review by sjwoodman — APPROVED** [review]

**Pull request merged**